gradle.properties

```diff
@@ -11,5 +11,7 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+android.useAndroidX=true
```

Comment:
What is this for?

Reply:
Android Extension Library --> All the libraries/packages apart from default android classes come under this. Example RecyclerView